src/general_manager/api/graphql.py

```
@@ -383,11 +383,16 @@ def _applyPermissionFilters(
         general_manager_class: type[GeneralManager],
         info: GraphQLResolveInfo,
     ) -> Bucket:
+        import copy
```

Ruff (0.12.2)

`copy` imported but unused

Remove unused import: `copy`

(F401)